ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11-periodics.yaml

```diff
@@ -16299,6 +16299,192 @@ periodics:
+    ci.openshift.io/generator: prowgen
+    job-release: "4.11"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-openshift-logging-5.5-azure-ipi-fips-p1-f3-openshift-logging-5-5
+  reporter_config:
+    slack:
+      channel: '#aos-qe-logging'
+      job_states_to_report:
+      - pending
+      - success
+      - failure
+      - aborted
+      - error
+      report_template: Job {{.Spec.Job}} state is *{{.Status.State}}* <{{.Status.URL}}|View
+        logs>
+  spec:
+    containers:
+    - args:
+      - --gcs-upload-secret=/secrets/gcs/service-account.json
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+      - --lease-server-credentials-file=/etc/boskos/credentials
+      - --oauth-token-path=/usr/local/github-credentials/oauth
+      - --report-credentials-file=/etc/report/credentials
+      - --secret-dir=/secrets/ci-pull-credentials
+      - --secret-dir=/usr/local/azure-ipi-fips-p1-f3-openshift-logging-5-5-cluster-profile
+      - --target=azure-ipi-fips-p1-f3-openshift-logging-5-5
+      - --variant=amd64-nightly-openshift-logging-5.5
+      command:
+      - ci-operator
+      image: ci-operator:latest
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 10m
+      volumeMounts:
+      - mountPath: /etc/boskos
+        name: boskos
+        readOnly: true
+      - mountPath: /secrets/ci-pull-credentials
+        name: ci-pull-credentials
+        readOnly: true
+      - mountPath: /usr/local/azure-ipi-fips-p1-f3-openshift-logging-5-5-cluster-profile
+        name: cluster-profile
+      - mountPath: /secrets/gcs
+        name: gcs-credentials
+        readOnly: true
+      - mountPath: /usr/local/github-credentials
+        name: github-credentials-openshift-ci-robot-private-git-cloner
+        readOnly: true
+      - mountPath: /etc/pull-secret
+        name: pull-secret
+        readOnly: true
+      - mountPath: /etc/report
+        name: result-aggregator
+        readOnly: true
+    serviceAccountName: ci-operator
+    volumes:
+    - name: boskos
+      secret:
+        items:
+        - key: credentials
+          path: credentials
+        secretName: boskos-credentials
+    - name: ci-pull-credentials
+      secret:
+        secretName: ci-pull-credentials
+    - name: cluster-profile
+      secret:
+        secretName: cluster-secrets-azure-qe
+    - name: github-credentials-openshift-ci-robot-private-git-cloner
+      secret:
+        secretName: github-credentials-openshift-ci-robot-private-git-cloner
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: result-aggregator
+      secret:
+        secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 00 21 * * 5,1,4
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: release-4.11
```

Comment:
release-4.13?

Reply:
Added release-4.14 profile for Logging 5.7. For the ginkgo branch, checking with test platform team as to why the generated job is using base_ref as release-4.11 even when we have specified the tag release-4.13 for openshift-tests-private image in the ci-operator config for the job.

Reply:
@anpingli @QiaolingTang I checked with Kui Wang for the Ginkgo branch config. The config we are using in this PR is correct. 

1 The cluster build is configured by [this config](https://github.com/openshift/release/pull/37208/files#diff-d08055a4adccc9bc31e46fc7bc9927cba29cad8051597b8a385bb84e0ddc70d2R40-R45), it decides we use 4.11 to create cluster.
2 The golang step take [image](https://github.com/openshift/release/pull/37208/files#diff-d08055a4adccc9bc31e46fc7bc9927cba29cad8051597b8a385bb84e0ddc70d2R18-R21) to create pod to start test. it is 4.13 branch image, so we take 4.13 test code to start cases.
3, [Our concern](https://github.com/openshift/release/pull/37208/files#diff-4ff7234a64bf60c6570775ca7f3d769feb9bde181bf8e5883f58b8ac8f3f5ba1R16402) is decided by [here](https://github.com/openshift/release/pull/37208/files#diff-d08055a4adccc9bc31e46fc7bc9927cba29cad8051597b8a385bb84e0ddc70d2R88-R91). it is correct for 4.11, we can not change it. even if we change it, it will be corrected by prow. it does not impact our test code. it just impact the image pipeline:root, pipeline:src, and pipeline:bin which are not used by golang test step.

I had ran [this](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/37208/rehearse-37208-periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-openshift-logging-5.7-aws-ipi-proxy-cco-manual-sts-p1-f7-openshift-logging-5-7/1635554621160165376) rehearsal which seems to have ran fine.

There is one more commit I need to add to this PR for the external hosted Grafana Loki instance. I'll add that today.

Reply:
Got it, thanks

Reply:
@QiaolingTang @anpingli Added the external grafana loki credential. Let me know if there is anything else needed as part of the PR and if we are good to merge it.